### ISSUE TITLE
How to install

### ISSUE THREAD

**Issue by schroef** [fiji/Stitching#44]
I came here by a link from ImageJ. Im wondering how and where i need to install this to use it.

Does this go in the main script folder or some other location?

**Comment by StephanPreibisch** [comment]
Hi, if you are not very experienced with Java I suggest using Fiji (http://fiji.sc) since it comes with all the packages you need.

**Comment by StephanPreibisch** [comment]
Otherwise you need to collect all the JARs that are listed in the dependency of this project and copy them to the ImageJ plugins directory: https://github.com/fiji/Stitching/blob/master/pom.xml They might have dependencies themselves, so this is maybe not the easiest way to it.

**Comment by ctrueden** [comment]
> They might have dependencies themselves, so this is maybe not the easiest way to it.

If you go that route, you can clone this project, then use `mvn dependency:copy-dependencies` to copy all needed JAR files into the `target/dependency` folder, then put all those JARs into ImageJ's `plugins` folder. But I second @StephanPreibisch's statement that this is not the easiest way. We have Fiji precisely so that users can avoid needing to do these things.

**Comment by schroef** [comment]
Yes, i found FIji and noticed this has it installed from start. I think the other method is to much for me. But collect JAR files and put them in a folder doesnt sound complicated at all. 

I was just missing some guidance as it lacks any info on that part, thanks again!

**Comment by ctrueden** [comment]
> I was just missing some guidance as it lacks any info on that part

Again: this lack of documentation is intentional, because it is a highly error-prone process. For example, if you follow this process for Stitching, and then again for another plugin which shares some of the same dependencies, you will end up with multiple versions of those dependencies in ImageJ's plugins folder and you will receive arcane error messages. Furthermore, [ImageJ2](https://imagej.net/ImageJ2) includes a copy of [ImageJ 1.x](https://imagej.net/ImageJ1), but with some augmentations, such that if you install a plugin intended for ImageJ2 into a plain ImageJ 1.x instance, you may encounter problems as well.

In short: this manual installation process is not recommended for users! We have [update sites](https://imagej.net/Update_Sites) for distribution of plugins.

If you do not want to use Fiji, it would be helpful to know your reasons so that we can work toward an ImageJ(2) which suits your needs in the future. (E.g.: some people complain that Fiji comes bundled with too many things—we have plans to make ImageJ2 more modular such that you can cherry-pick which plugins you want to install more easily. It is possible to do this now via the [Updater](https://imagej.net/Updater) user interface, but more of a hassle than it should be.)
